keras/layers/convolutional.py

```diff
@@ -2040,7 +2040,50 @@ def get_config(self):
         return dict(list(base_config.items()) + list(config.items()))
 
 
-class ZeroPadding1D(Layer):
+class _ZeroPadding(Layer):
```

Comment:
If this base layer does not have a `call` method, it should implement `call` and raise `NotImplementedError` there.